### DISCUSSION TITLE
QEMU suggestions

### DISCUSSION THREAD

**Discussion by bol-van** [Genymobile/feedback#77] [Genymotion Desktop] [answered]
Pls add "-cpu host" parameter to qemu-system-x86-64.  By default qemu disables all advanced CPU instruction sets.
Pls add ability to disable snapshots. On my system they are not working. Every time it boots from the beginnig. But snapshots take lot of time on HDD or waste SSD resource
Pls add bridge support to QEMU

**Comment by agavignet** [comment]
Hi @bol-van,

Thanks for your feedback.

> Pls add "-cpu host" parameter to qemu-system-x86-64. By default qemu disables all advanced CPU instruction sets.

We will add it and push it in the next Desktop version if the QA is approved on all our platforms. In the meantime, you can set the env var `GM_QEMU_START_ARGS="-cpu=host"` as a workaround.

> Pls add ability to disable snapshots.

We will consider adding this possibility in the next Desktop version too.

Thanks again,
Adrien

**Comment by agavignet** [comment, marked as answer]
Hi @bol-van 

Your first two suggestions have been taken into account in Desktop 3.6 released today. Thank you :)

Adrien
